src/libraries/System.Private.CoreLib/src/System/Threading/ThreadPool.Portable.cs

```diff
@@ -31,6 +31,9 @@ public static partial class ThreadPool
             AppContextConfigHelper.GetBooleanConfig("System.Threading.ThreadPool.EnableWorkerTracking", false);
 #endif
 
+        internal static readonly bool EnableDispatchAutoReleasePool =
```

Comment:
@lambdageek should this be always true for ios/tvos ?

Reply:
@marek-safar @jkoritzinsky yes, this should also be true for tvOS and iOS